packages/blade/src/components/Card/CardHeader.tsx

```diff
@@ -0,0 +1,194 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import React from 'react';
+import { Divider } from './Divider';
+import { useInsideCard } from './CardContext';
+import Box from '~components/Box';
+import type { TextProps, TextVariant } from '~components/Typography';
+import { Heading, Text } from '~components/Typography';
+import type { IconComponent } from '~components/Icons';
+import type { CounterProps } from '~components/Counter';
+import { Counter } from '~components/Counter';
+import type { BadgeProps } from '~components/Badge';
+import { Badge } from '~components/Badge';
+import type { LinkProps } from '~components/Link';
+import { Link } from '~components/Link';
+import type { ButtonProps } from '~components/Button';
+import { Button } from '~components/Button';
+import { minHeight } from '~components/Button/BaseButton/buttonTokens';
+import { makeSpace } from '~utils';
+
+type CardHeaderProps = {
```

Comment:
nit: move this type close to the component like other types? i was scrolling up and down to locate for it :D

Reply:
Yeah sure, I added that closer but then added other components and pushed it up xD